<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerifyMnemonicSeedDialog</class>
 <widget class="QDialog" name="VerifyMnemonicSeedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>231</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New deterministic wallet created</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>This is mnemonic seed of your wallet. Write it down carefully. It allows to restore your wallet.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mnemonic:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTextEdit" name="m_seedEdit">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Repeat:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="m_seedRepeat">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_languageLabel">
       <property name="text">
        <string>Select language</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_languageCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_okButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_okButton</sender>
   <signal>clicked()</signal>
   <receiver>VerifyMnemonicSeedDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_seedRepeat</sender>
   <signal>textChanged()</signal>
   <receiver>VerifyMnemonicSeedDialog</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_languageCombo</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>VerifyMnemonicSeedDialog</receiver>
   <slot>languageChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectPathClicked()</slot>
 </slots>
</ui>
